inc/theme/namespace.php

```diff
@@ -130,6 +130,20 @@ function migrate_book_themes() {
 		}
 
 		update_option( 'pressbooks_theme_migration', 1 );
+	} elseif ( get_option( 'pressbooks_theme_migration' ) === 1 ) {
```

Comment:
I'm not sure it should be `elseif` here.

This works for existing books, or books already migrated to state 1, but on a new book or a book that has been in idle mode for many months getting into this `elseif` condition will require two page loads. (Per your unit tests the first page load will set `pressbooks_theme_migration` to 1, the second to 2.)

Maybe a separate `if` condition so that if a migration happens then it happens all at once?